example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
```

Comment:
My bad for putting this into version control

Reply:
This should be fixed!